.phpcs.xml.dist

```diff
@@ -1,7 +1,7 @@
 <?xml version="1.0"?>
 <ruleset name="Jetpack">
 	<config name="minimum_supported_wp_version" value="5.0" />
-	<config name="testVersion" value="5.2-"/>
+	<config name="testVersion" value="5.6-"/>
```

Comment:
I know I suggested this, but we'll need to get this down to 5.3 for the time being, see https://github.com/Automattic/jetpack/pull/12379#issuecomment-492773072